src/main/java/com/alkemy/ong/controller/TestimonialController.java

```diff
@@ -84,8 +82,7 @@
 
   @PostMapping(produces = {"application/json"},
       consumes = {"application/json"})
-  @ApiOperation(value = "Create a testimonial", produces = "application/json",
-      consumes = "application/json")
+  @ApiOperation(value = "Create a testimonial", produces = "application/json")
```

Comment:
Está operación consume and produce son JSON porque son el tipo de entrada y salida de la API.